.circleci/config.yml

```diff
@@ -60,16 +58,14 @@ jobs:
           name: Link Sequences
           command: npm run deploy-routes
       - helix-post-deploy/monitoring:
-          targets: universal, aws
+          targets: aws
```

Comment:
```suggestion
          targets: universal, aws
```

Reply:
none of our services is deployed to universal anymore. so I would remove it here as well.

Reply:
The published service URL and the service URL that is being used by other services is on anywhere.run, so the removal would prevent new functionality or bug fixes from being used.